docs/body_file.md

```diff
@@ -8,138 +8,156 @@ tags:
-`Type: $FILE_NAME (48-3)   Name: N/A   Resident   size: 82`
-`Type: $DATA (128-2)   Name: N/A   Resident   size: 0`
-`Type: $DATA (128-1)   Name: $Bad   Non-Resident   size: 7270400  init_size: 0`
+Default output fields are as follows:
 
-Behavior of the TSK Metadata Address:
+### Version 2.x
 
-- 8-128-2 references the \$DATA attribute in MFT entry 8, note that this
-  \$DATA attribute on-disk is the 3rd attribute in the MFT entry.
-- 8-128-1 references the \$DATA attribute in MFT entry 8, note that this
-  \$DATA attribute on-disk is the 4th attribute in the MFT entry.
-- 8-48-3 references the first \$FILE_NAME attribute in MFT entry 8, note
-  that the \$FILE_NAME attribute on-disk is the 2nd attribute in the MFT
-  entry.
+```ascii
+MD5 | path/name | device | inode | mode_as_value | mode_as_string | num_of_links | UID | GID | rdev | size | atime | mtime | ctime | block_size | num_of_blocks
+```
+
+### Version 3.x
+
+```ascii
+MD5 | name | inode | mode_as_string | UID | GID | size | atime | mtime | ctime | crtime
+```
+
+_Note: Outputs do not include spaces between values and pipes; we have inserted them here for easier readability._
 
-From the body file:
-
-`0|/$Extend|11-144-4|d/dr-xr-xr-x|0|0|552|1580550524|1580550524|1580550524|1580550524`
-`0|/$Extend ($FILE_NAME)|11-48-3|d/dr-xr-xr-x|0|0|80|1580550524|1580550524|1580550524|1580550524`
-
-Corresponding MFT entry:
-
-`MFT Entry Header Values:`
-`Entry: 11        Sequence: 11`
-`$LogFile Sequence Number: 1070011`
-`Allocated Directory`
-`Links: 1`
-``
-`$STANDARD_INFORMATION Attribute Values:`
-`Flags: Hidden, System`
-`Owner ID: 0`
-`Security ID: 257  ()`
-`Created:    2020-02-01 10:48:44.857384500 (CET)`
-`File Modified:  2020-02-01 10:48:44.857384500 (CET)`
-`MFT Modified:   2020-02-01 10:48:44.857384500 (CET)`
-`Accessed:   2020-02-01 10:48:44.857384500 (CET)`
-``
-`$FILE_NAME Attribute Values:`
-`Flags: Directory, Hidden, System`
-`Name: $Extend`
-`Parent MFT Entry: 5     Sequence: 5`
-`Allocated Size: 0       Actual Size: 0`
-`Created:    2020-02-01 10:48:44.857384500 (CET)`
-`File Modified:  2020-02-01 10:48:44.857384500 (CET)`
-`MFT Modified:   2020-02-01 10:48:44.857384500 (CET)`
-`Accessed:   2020-02-01 10:48:44.857384500 (CET)`
-``
-`Attributes: `
-`Type: $STANDARD_INFORMATION (16-0)   Name: N/A   Resident   size: 72`
-`Type: $FILE_NAME (48-3)   Name: N/A   Resident   size: 80`
-`Type: $INDEX_ROOT (144-4)   Name: $I30   Resident   size: 552`
+---
+
+## Known Issues
+<!-- Issues have been merged from multiple files. Also, issues may apply to The Sleuth Kit more than body files - must reconcile. 
+NOTE: Nov/Dec 2022, validate if each is still an issue -->
+Known shortcomings with body file format are:
+
+* Undocumented granularity of timestamp, current implementation by
+  [The Sleuth Kit](sleuthkit.md) appears to be seconds. See [here](https://github.com/sleuthkit/sleuthkit/issues/1810).
+* Undocumented extended file mode of `-/-rwxrwxrwx`.
```

Comment:
maybe reword this to make it clear that both `-/-` and the leading `r` in `rrwxrwxrwx` are undocumented